rootfs/api/models/build.py

```diff
@@ -43,10 +43,11 @@ def create(self, user, *args, **kwargs):
         try:
             self.app.deploy(new_release)
             return new_release
-        except Exception:
+        except Exception as e:
             if 'new_release' in locals():
                 new_release.delete()
-            raise
+
+            raise DeisException(str(e)) from e
```

Comment:
I think that's the wrong way to do it and lends for a bad user experience. If you are very concerned by this and the nicer UX then we can look at a custom exception handler that can log more ifnromation about parent exception, since I am attaching the originating exception to the re-raised one